src/main/scala/mesosphere/marathon/event/http/HttpEventStreamActor.scala

```diff
@@ -16,49 +22,117 @@ trait HttpEventStreamHandle {
   def close(): Unit
 }
 
+class HttpEventStreamActorMetrics @Inject() (metrics: Metrics) {
+  val numberOfStreams: AtomicIntGauge =
+    metrics.gauge(metrics.name("api", getClass, "number-of-streams"), new AtomicIntGauge)
```

Comment:
I would like to have a more specific method like metrics.name -> metrics.api or something like that. So that we don't deal with strings here.